conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/Lorre.scala

```diff
@@ -153,6 +153,48 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
     }
   }
 
+  /** Updates bakers in the DB */
+  def updateBakers(): Future[Unit] = {
```

Comment:
According to the ticket, the bakers are supposed to be updated once per cycle. Could you point me to the line of code which checks whether we have entered a new cycle and only then triggers a bakers update?

Reply:
I've misunderstood meaning of the cycle. I've updated PR so the last cycle will be passed to the next execution and then compared.

Reply:
We previously outlined some logic for you to figure out the start of a new cycle by doing some math on the current block level, e.g. something like taking the modulo of the block number by 4096. Are you doing that here as well?

Reply:
so, because we are doing it in outside of main processing, we cant now if after break we processed single block or few blocks - we don't have a way to be sure that next time we will try to run our update we will have block head equal for example 0. Let's say we set our check to find level mod 4096 == 0, and our current level after end of processing the batch is 4095. Lorre "goes to sleep" and next time processes 2 blocks. This way we have no way of be sure that Lorre will update bakers during this cycle.

In this case I'm simply passing last processed cycle as parameter, and check if current latest stored in the DB i greater than the one passed.